source/georef/georef.cpp

```diff
@@ -615,7 +615,7 @@ std::vector<Admin*> GeoRef::find_admins(const type::GeographicalCoord& coord, Ad
     if(!result.empty()){
         return result;
     }
-    //we didn't found any results with boundary, as a fallback we search for the admin of the closest way
+    //we didn't find any result with boundary, as a fallback we search for the admin of the closest way
```

Comment:
we didn't find any result with**in the** boundary, as a fallback we search for the **nearest** admin.